great-idea/css/index.css

```diff
@@ -63,4 +63,227 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+	width: 880px;
+	/* TB LR */
+	margin: 0 auto;
+  }
+  
+  /* header */
+  
+  header {
+	display: flex;
+  }
+  header nav {
+	width: 691px;
+	vertical-align: top;
+	margin-top: 35px;
+  }
+  
+  header nav a {
+	margin-right: 65px;
+	font-size: 14px;
+	color: gray;
+	text-decoration: none;
+  
+  }
+  
+  header nav a:hover {
+	text-decoration: underline;
+  }
+  
+  header nav a:first-child {
+	margin-left: 0;
+  }
+  
+  header nav a:last-child {
+	margin-right: 0;
+  }
+  
+  header .logo {
+	margin-top: 25px;
+  }
+  
+  /* cta */
+  
+  .cta {
+	display: flex;
+  }
+  .cta .cta-text {  
+	vertical-align: top;
+	font-size: 77px;
+	margin-top: 120px;
+	text-align: center;
+	padding: 0px 75px;
+  }
+  
+  .cta .cta-text button {
+	background: white;
+	vertical-align: top;
+	margin-top: 6px;
+	padding: 8px 44px;
+	font-size: 17px;
+	border: 1px solid black;
+	cursor: pointer;
+  }
+  
+  .cta .cta-text button:hover {
+	background: black;
+	color: white;
+  }
+  
+  .cta img {
+	margin-top: 69px;
+  }
+  
+  /* main */
+   
+  .main-content {
+	
+	border-top: 1px solid black;
+	
+	margin-top: 76px;
+	padding-top: 39px;
+  }
+  
+  .main-content .top-content {
+	display: flex;
+	padding: 0 10px;
+	
+  }
+  
+  .main-content .top-content .text-content {
+	width: 50%;
+	 margin-right: 34px;
+	 
+  }
+  
+  .main-content .top-content .text-content:last-child {
+	margin-right: 0;
+	
+  }
+  
+  .main-content .middle-img {
+	margin-top: 45px;
+	
+  }
+  
+  .main-content .bottom-content {
+	display: flex;
+	margin-top: 37px;
+	padding: 0 10px;
+	
+  }
+  
+  .main-content .bottom-content .text-content {
+	width: 275px;
+	margin-right: 12px;
+	
+  }
+  
+  .main-content .bottom-content .text-content:last-child {
+	margin-right: 0;
+	
+  }
+  
+  /* contact */
+  .contact {
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+	margin-top: 57px;
+	border-top: 1px solid black;
+	padding: 39px 10px 0;
+  }
+  
+  .contact p {
+	margin: 20px 0;
+  }
+  
+  /* footer */
+  
+  footer {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+	margin: 32px 0;
+	text-align: center;
+	font-size: 14px;
+	}
+	/* Services Page */
+
+	.services-header-img {
+		margin: 20px 0;
+	}
+
+	.our-services {
+		
+		margin: 20px 0;-
+		border-bottom: 1px solid black;
```

Comment:
It looks like somewhere around here there's an indentation change.  Try downloading an extension for format your code to keep it consistent.  eslint and prettier are great choices